lib/core/test/unit/Cardano/Wallet/DBSpec.hs

```diff
@@ -387,8 +387,8 @@ instance Eq XPrv where
 genRootKeys :: Gen (Key 'RootK XPrv)
 genRootKeys = do
     (s, g, e) <- (,,)
-        <$> genPassphrase @"seed" (0, 32)
-        <*> genPassphrase @"generation" (0, 16)
+        <$> genPassphrase @"seed" (16, 32)
+        <*> genPassphrase @"generation" (10, 16)
```

Comment:
Why the second change to `10` here?

Reply:
because of that - https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs#L246